pynetdicom3/sop_class.py

```diff
@@ -591,7 +591,7 @@ def SCP(self, msg):
         # Build C-FIND response primitive
         c_find_rsp = C_FIND()
         c_find_rsp.MessageIDBeingRespondedTo = msg.MessageID
-        c_find_rsp.AffectedSOPClassUID = msg.AffectedSOPClassUID
+        sop_class = c_find_rsp.AffectedSOPClassUID = msg.AffectedSOPClassUID
```

Comment:
I don't think this is necessary?

Reply:
Yeah, you're right; I probably added this before deciding to pass the entire instance (see below).